crates/subspace-verification/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright (C) 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
I don't think Parity has much to do with the code, I wrote all the code moved here, it is all Subspace-specific